schema_utils_test.py

```diff
@@ -433,6 +433,55 @@ def test_schemas_are_correctly_validated(self):
             with self.assertRaises((AssertionError, KeyError)):
                 validate_schema(schema)
 
+    def test_is_regex_raises_not_implemented_error(self):
+        """Test if is_regex(obj) raises NotImplementedError."""
+        with self.assertRaises(NotImplementedError):
+            schema_utils._Validators.is_regex('string') # pylint: disable=protected-access
+
+    def test_matches_regex_raises_not_implemented_error(self):
+        """Test if matches_regex(obj) raises NotImplementedError."""
+        with self.assertRaises(NotImplementedError):
+            schema_utils._Validators.matches_regex('s1', 's2') # pylint: disable=protected-access
+
+    def test_normalize_against_schema_raises_exception(self):
+        """Tests if normalize against schema raises exception
+        for invalid key.
+        """
+        with self.assertRaises(Exception):
+            schema = {SCHEMA_KEY_TYPE: 'invalid'}
+            schema_utils.normalize_against_schema('obj', schema)
+
+    def test_is_nonempty(self):
+        """Tests if static method is_nonempty returns true iff obj
+        is not empty str.
+        """
+        is_nonempty = schema_utils._Validators.get('is_nonempty') # pylint: disable=protected-access
```

Comment:
Here and below: call this is_nonempty_validator or validator or something with validator in the name.

Reply:
addressed in next commit/PR